values.md

```diff
@@ -1,36 +1,36 @@
+The values of the Rockefeller Archive Center's Digital Program inform how we work together as a team, as well as how we interact with our RAC colleagues, researchers, donors, and other external partners. They support and shape our strategic goals and provide a standard by which we measure our daily work. These values are based in an understanding of technologies as embedded in and shaped by human biases, histories of oppression, and power. Consequently, our values exist to make us _a part of_ a community of practice that engages technology from a basis of values and care, not to set us _apart from_ our colleagues, researchers, donors, or other external partners.
 
-We respect and contribute to a vibrant culture based on these values, and raise concerns clearly when they are not being followed by members of the Digital Programs team. We review our values on an annual basis and revise them as necessary.
+We review our values on an annual basis and revise them as necessary.
 
 ## Engage and empower users.
 
-We serve multiple constituencies and always work for the good of others. Engage with users to understand their needs, prioritize solutions, and produce outcomes that provide high-quality and useful services for researchers, donors and RAC staff. Provide the care, resources, and training necessary for users to feel empowered to use and troubleshoot systems. Approach work with empathy and humility: listening before solving, valuing the work of others, and recognizing that we still have room to learn.
+We engage directly with multiple user communities - including researchers, donors, and our colleagues at the RAC - to understand their needs, prioritize solutions, and provide high-quality and useful services and products. In every aspect of our work, we are committed to providing the care, resources, and training necessary to empower users as they interact with systems. As a result, we aspire to approach our work with empathy and humility: listening before solving, valuing the labor and expertise of others, and recognizing that we still have room to learn.
 
 ## Provide ethical access
 ### Respect researchers' privacy.
 
-Support remote and on-site researchers' right to privacy and confidentiality. Advocate for sensible data retention policies and gathering minimal data.
+We advocate for our researchers' right to privacy, confidentiality and self-determination in both on-site and remote research environments. When implementing systems, we seek to minimize the amount of user data gathered and stored, and provide mechanisms to facilitate sensible data retention policies. By improving the usability and accessibility of systems, we facilitate independent, self-directed access, and improve users’ ability to make choices about their personal data.
```

Comment:
I think it's important to have self-determination explicitly mentioned in our values, but I'm wondering about the thinking of including it in this section as opposed to the following. I think it could fit in either--but I'm wondering if this section heading should be updated.

Reply:
This was one that @HaSistrunk and I went back and forth on for a while. I'd prefer to have her jump in here because I think she's got the most fully-developed analysis of this particular area.

Reply:
This was an attempt to tie self-determination (via usability/accessibility) explicitly to privacy. If the system is usable, users have more control about when they involved someone else and for what. Does that make sense?

Reply:
@HaSistrunk I'm not sure I totally understand what you're getting at with the above comment, but I also don't have a strong opinion to include it in one section over the other. I'm basically commenting because I'd like to hear you talk more about what it means to have it in this section so I better understand how our values fit together. That conversation doesn't have to be on GH though.

Reply:
OK I've done three things here:
- update the heading for this section to be a bit broader
- rework the language for the section below (broad and equitable access) to be a bit more direct
- replace one of the "facilitates" with "promotes"

Thoughts?